src/openrct2/entity/Peep.h

```diff
@@ -311,8 +320,16 @@ struct Peep : EntityBase
     PeepAnimationGroup AnimationGroup;
     uint8_t TshirtColour;
     uint8_t TrousersColour;
-    uint16_t DestinationX; // Location that the peep is trying to get to
-    uint16_t DestinationY;
+    union
+    {
+        uint16_t DestinationX; // Location that the peep is trying to get to
```

Comment:
This comment adds 0.0 information. There is no point stating the bleeding obvious.

```suggestion
        uint16_t DestinationX;
```

Reply:
The comment was already there before, I didn't add it